README.md

```diff
@@ -34,7 +34,7 @@ instructions [here][create-key] to create a service account, generate a private
 key, and download a JSON file containing the private key. Place the JSON file on
 the machine that will be mounting the bucket.
 
-[console]: console.developers.google.com
+[console]: https://console.developers.google.com/project/_/storage/browser
```

Comment:
Totally striking out here. I meant this as a help
https://console.developers.google.com/project/_/apiui/credential - which could get the user to pick the project and then dumpt them on the credential page if I understood the intent.